deploy.sh

```diff
@@ -24,7 +25,8 @@ if [ "$registry" != "remote" ] && [ "$registry" != "local" ]; then
     exit 2
 fi
 
-if cd deploy/$1 && kubectl apply -f crd.yaml && kubectl apply -f rbac.yaml && kubectl apply -f deploy_$registry.yaml; then
+## use imperative management for CRDs since metadata for PodTemplateSpec is too long.
+if cd deploy/$1 && kubectl create -f crd.yaml && kubectl apply -f rbac.yaml && kubectl apply -f deploy_$registry.yaml; then
```

Comment:
@codyjrivera just curious, why use `create` rather than `apply` here?

Reply:
The `template: PodTemplateSpec` field of the `VStatefulSet` spec makes the resulting CRD file too large to work with the declarative `kubectl apply`. I admit that using the imperative `kubectl create` is a cheap workaround. Is there an obviously better way of doing this?